src/main/java/frc/robot/services/ArmGravityCompensation.java

```diff
@@ -0,0 +1,70 @@
+package frc.robot.services;
+
+import com.spikes2212.dashboard.RootNamespace;
+import frc.robot.subsystems.ArmFirstJoint;
+import frc.robot.subsystems.ArmSecondJoint;
+
+import java.util.function.Supplier;
+
+import static frc.robot.subsystems.ArmFirstJoint.GEAR_RATIO_MOTOR_TO_ABSOLUTE_ENCODER;
+
+/**
+ * Adds the voltage needed to defy gravity to the two joints of the arm.
```

Comment:
Change to: "Adds the voltage needed for the two joints of the arm to defy gravity."